x-pack/platform/plugins/shared/security/public/management/role_mappings/edit_role_mapping/rule_editor_panel/rule_group_editor.test.tsx

```diff
@@ -25,8 +31,8 @@ describe('RuleGroupEditor', () => {
       onChange: jest.fn(),
       onDelete: jest.fn(),
     };
-    const wrapper = shallowWithIntl(<RuleGroupEditor {...props} />);
-    expect(wrapper.find(RuleGroupEditor)).toHaveLength(0);
+    const wrapper = mountWithEuiProvider(<RuleGroupEditor {...props} />);
```

Comment:
Note: now that it is nested in the EuiProvider, the first assertion is finding the element that was passed in. 

So just one is found, no sub RuleGroupEditors